### ISSUE TITLE
Manual, 20.1. Synapse Homeserver: example is wrong

### ISSUE THREAD

**Issue by loewenheim** [NixOS/nixpkgs#85327]
The example configuration suggests the setting
```
 services.postgresql.initialScript = ''
    CREATE ROLE "matrix-synapse" WITH LOGIN PASSWORD 'synapse';
    CREATE DATABASE "matrix-synapse" WITH OWNER "matrix-synapse"
      TEMPLATE template0
      LC_COLLATE = "C"
      LC_CTYPE = "C";
  '';
```
This fails because `services.postgresql.initialScript` has to be of type `null or path`.

**Comment by worldofpeace** [comment]
@ma27 (I think)

**Comment by Ma27** [comment]
Thanks for pointing this out! Fixed on `master` and `release-20.03` :)
